docs/mevblocker/concepts/order-flow-auction.md

```
@@ -0,0 +1,67 @@
+---
+sidebar_position: 1
+---
+
+# How it works
+
+MEV Blocker's transaction flow can be broken down into 6 main steps. The image below shows a general overview of the flow of a transaction.
+
+![The Best Protection Under the Sun](/img/mevblocker/mevblocker_ofa.png)
+
+### 1. Builders pay subscription fee and connect to MEV Blocker
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)